components/AnimationTiming/src/CAMediaTimingFunction+MDCAnimationTiming.m

```diff
@@ -19,13 +19,13 @@ @implementation CAMediaTimingFunction (MDCAnimationTiming)
 + (CAMediaTimingFunction *)mdc_functionWithType:(MDCAnimationTimingFunction)type {
   switch (type) {
     case MDCAnimationTimingFunctionStandard:
-      return [[CAMediaTimingFunction alloc] initWithControlPoints:(float)0.4:0:(float)0.2:1];
+      return [[CAMediaTimingFunction alloc] initWithControlPoints:(float) 0.4:0:(float)0.2:1];
```

Comment:
This file should be reverted.  Should probably add `NOLINT` markers to all calls of `initWithControlPoints`